Library/Homebrew/cask/lib/hbc/download_strategy.rb

```diff
@@ -181,7 +181,7 @@ def referer_args
     end
 
     def ext
-      Pathname.new(@url).extname
+      Pathname.new(@url).extname.sub(/\?.*/, '')
```

Comment:
Thank you @reitermarkus for the review. I've updated the PR. Please have a look.